doc/source/migrate_from_ppa.md

```diff
@@ -0,0 +1,89 @@
+:orphan:
+
+# Migration from PPA (Personal Package Archive) to Groonga APT Repository (packages.groonga.org)
+
+This document guides you through the migration from the deprecated PPA (Personal
+Package Archive) to our new Groonga APT Repository (packages.groonga.org).
+
+By switching to the APT Repository, you'll receive packages built with Apache
+Arrow enabled. Which unlocks extra features such as parallel offline index
+building. We strongly recommend you to migrate to enjoy these enhancements.
+
+If you are currently using the old PPA (ppa:groonga/ppa), please follow the
+steps below to switch to the new package source.
+
+## Configure the Groonga APT Repository
+
+### Register Groonga APT Repository (packages.groonga.org)
+
+To register the Groonga APT repository, install the `groonga-apt-source package`
+as follows.
+
+```bash
+sudo apt update
+sudo apt install -y -V groonga-apt-source
+sudo apt update
+```
+
+### Remove the Existing Package Source
+
+Remove the old Groonga PPA (ppa:groonga/ppa):
+
+```bash
+sudo add-apt-repository -y --remove ppa:groonga/ppa
+sudo apt update
+```
+
+After these steps, your package source is now switched from PPA
+(ppa:groonga/ppa) to APT Repository (packages.groonga.org).
+
+## Package Installation
+
+There are two approaches depending on your needs.
+
+### Option 1: Use Groonga APT Repository to Install the Next Package Version (Recommended)
+
+Starting with the next package version, we strongly recommend using the Groonga
+APT Repository. When you install Groonga via APT, you will automatically receive
+the next package version without having to specify a version explicitly as
+follows. This option is simpler and is recommended for most users:
+
+```bash
+sudo apt install package-name
+```
+
+For example, install groonga package:
+
+```bash
+sudo apt install groonga
```

Comment:
`groonga-bin` may be better than `groonga`.
Because `groonga` depends on `groonga-doc` that isn't needed for many users.

Reply:
fix: [768b6ad](https://github.com/groonga/groonga/pull/2260/commits/768b6ad2882f37b83320e1f4fbf33a1f82606976) Sure.